uint8 status
Header header
float64[2] inputVec